aries-site/src/pages/templates/code-blocks.mdx

```diff
@@ -0,0 +1,54 @@
+import { Example } from '../../layouts';
+import {
+  CodeBlockExample
+} from '../../examples';
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/CodeBlocks.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/DarkCodeTheme.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/LightCodeTheme.js",
+  ]}
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <CodeBlockExample />
+</Example>
+
+
+## Guidance
+
+Code blocks should be created using the [react-syntax-highlighter](https://github.com/react-syntax-highlighter/react-syntax-highlighter) tool.
+
+Prism or Highlighter.js can be used with react-syntax-highlighter with Highlighter.js being the default. Highlighter.js occasionally has trouble highlighting jsx so it is recommended to use Prism instead. 
+The documentation for using Prism with react-syntax-highlighter includes a [list of available languages](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/AVAILABLE_LANGUAGES_PRISM.MD).
+
+react-syntax-highlighter can be installed with the following command:
+
+`yarn add react-syntax-highlighter`
+
+Import the react-syntax-highlighter specifying Prism as the highlighter with the following:
+
+`import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';`
+
+### Theming
+
+The following themes should be used to style code blocks.
+
+[Light mode theme](https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/DarkCodeTheme.js)
```

Comment:
are we going to change these links to point to the new theme files or are we leaving these files in here as well?

Reply:
sounds good! Looks Good other than that!!